### PR TITLE
meagerfindings ergodox

### PR DESCRIPTION
**Pull request by meagerfindings**
Initial commit of Ergodox layout, finally....

### CODE REVIEW

File: layouts/community/ergodox/meagerfindings/LICENSE [hunk 1]
```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (drashna) [inline layouts/community/ergodox/meagerfindings/LICENSE]: @jackhumbert Is this necessary?

File: layouts/community/ergodox/meagerfindings/config.h [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#ifndef CONFIG_MEAGERFINDINGS_H
```
Comment (drashna) [inline layouts/community/ergodox/meagerfindings/config.h]: If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?

File: layouts/community/ergodox/meagerfindings/config.h [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+#define CONFIG_MEAGERFINDINGS_H
+
+#include QMK_KEYBOARD_CONFIG_H
```
Comment (drashna) [inline layouts/community/ergodox/meagerfindings/config.h]: Could you remove this? 

It doesn't actually do anything.

File: layouts/community/ergodox/meagerfindings/config.h [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+#undef MOUSEKEY_DELAY
+#define MOUSEKEY_DELAY          0
+#undef MOUSEKEY_TIME_TO_MAX
+#define MOUSEKEY_TIME_TO_MAX    18
+#undef MOUSEKEY_MAX_SPEED
+#define MOUSEKEY_MAX_SPEED      4
+#undef MOUSEKEY_WHEEL_DELAY
+#define MOUSEKEY_WHEEL_DELAY 0
+
+#define PREVENT_STUCK_MODIFIERS
```
Comment (drashna) [inline layouts/community/ergodox/meagerfindings/config.h]: Could you remove this?  It's the new default, now, actually.

File: layouts/community/ergodox/meagerfindings/keymap.c [hunk 5]
```diff
@@ -0,0 +1,566 @@
+  #include QMK_KEYBOARD_H
+#include "debug.h"
```
Comment (drashna) [inline layouts/community/ergodox/meagerfindings/keymap.c]: Could you removal all but the `QMK_KEYBOARD_H` and the version.h includes? 

The rest should be included already.

File: layouts/community/ergodox/meagerfindings/keymap.c [hunk 6]
```diff
@@ -0,0 +1,566 @@
...
+      _______,     KC_EQUAL,      KC_7,          KC_8,        KC_9,        KC_ASTR,        _______,
+                    KC_PLUS,      KC_4,          KC_5,        KC_6,        KC_PLUS,        _______,
+      _______,     KC_MINUS,      KC_1,          KC_2,        KC_3,        KC_KP_SLASH,    _______,
+                    KC_KP_0,      KC_DOT,        DBLE_ZER0,   DBLE_ZER0,   KC_KP_ENTER,
+      _______,      _______,
+      _______,
+      _______,      KC_PENT,      _______),
+};
+
+const uint16_t PROGMEM fn_actions[] = {
```
Comment (noroadsleft) [inline layouts/community/ergodox/meagerfindings/keymap.c]: This function is deprecated and should be removed.

Your keymap doesn't use this functionality at all.

File: layouts/community/ergodox/meagerfindings/LICENSE (hunk 1 above)
Comment (meagerfindings) [reply layouts/community/ergodox/meagerfindings/LICENSE]: Removing!

File: layouts/community/ergodox/meagerfindings/config.h (hunk 3 above)
Comment (meagerfindings) [reply layouts/community/ergodox/meagerfindings/config.h]: Removed

File: layouts/community/ergodox/meagerfindings/config.h (hunk 4 above)
Comment (meagerfindings) [reply layouts/community/ergodox/meagerfindings/config.h]: Removed

File: layouts/community/ergodox/meagerfindings/keymap.c (hunk 5 above)
Comment (meagerfindings) [reply layouts/community/ergodox/meagerfindings/keymap.c]: Removed

File: layouts/community/ergodox/meagerfindings/keymap.c (hunk 6 above)
Comment (meagerfindings) [reply layouts/community/ergodox/meagerfindings/keymap.c]: Removed

File: layouts/community/ergodox/meagerfindings/config.h (hunk 2 above)
Comment (yiancar) [reply layouts/community/ergodox/meagerfindings/config.h]: There is no need for you to undefine a define if your only goal is to redefine it again.

Also just to make your code a bit prettier, some stuff have default defines eg USB_MAX_POWER_CONSUMPTION  has a default value of 500, so no need to define it again:)

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**